test/redeem-script-parser-test.js

```diff
@@ -83,66 +83,38 @@ describe('getPowpegRedeemScript', () => {
+        expect(() => redeemScriptParser.getP2shErpRedeemScript(publicKeys, p2shErpBtcPublicKeys, null)).to.throw(ERROR_MESSAGES.INVALID_CSV_VALUE);
+        expect(() => redeemScriptParser.getP2shErpRedeemScript(publicKeys, p2shErpBtcPublicKeys, '')).to.throw(ERROR_MESSAGES.INVALID_CSV_VALUE);
     });
 
-    it('should return a valid erp redeem script', () => {
-        let erpRedeemScript = redeemScriptParser.getErpRedeemScript(publicKeys, erpPublicKeys, csvValue).toString('hex');
-        validateErpRedeemScriptFormat(erpRedeemScript, publicKeys, erpPublicKeys);
-    });
-});
-
-describe('getFlyoverRedeemScriptFromPublicKeys', () => {
-    const dHash = crypto.randomBytes(32).toString('hex');
-    const publicKeys = [getRandomPubkey(), getRandomPubkey(), getRandomPubkey()];
-    
-    it('should fail for invalid data', () => {    
-        // fail because there is no derivation hash
-        expect(() => redeemScriptParser.getFlyoverRedeemScriptFromPublicKeys(null)).to.throw(ERROR_MESSAGES.INVALID_DHASH);
-        expect(() => redeemScriptParser.getFlyoverRedeemScriptFromPublicKeys(null, null)).to.throw(ERROR_MESSAGES.INVALID_DHASH);
-        expect(() => redeemScriptParser.getFlyoverRedeemScriptFromPublicKeys(publicKeys, null)).to.throw(ERROR_MESSAGES.INVALID_DHASH);
-        expect(() => redeemScriptParser.getFlyoverRedeemScriptFromPublicKeys(publicKeys, '')).to.throw(ERROR_MESSAGES.INVALID_DHASH);
-        // a short hash
-        expect(() => redeemScriptParser.getFlyoverRedeemScriptFromPublicKeys(publicKeys, dHash.substring(1))).to.throw(ERROR_MESSAGES.INVALID_DHASH);
-        // a long hash
-        expect(() => redeemScriptParser.getFlyoverRedeemScriptFromPublicKeys(publicKeys, dHash.concat('1'))).to.throw(ERROR_MESSAGES.INVALID_DHASH);
-        
-        // fail because there are no public keys
-        expect(() => redeemScriptParser.getFlyoverRedeemScriptFromPublicKeys(null, dHash)).to.throw(ERROR_MESSAGES.INVALID_POWPEG_PUBLIC_KEYS);
-        expect(() => redeemScriptParser.getFlyoverRedeemScriptFromPublicKeys('', dHash)).to.throw(ERROR_MESSAGES.INVALID_POWPEG_PUBLIC_KEYS);
-        expect(() => redeemScriptParser.getFlyoverRedeemScriptFromPublicKeys(getRandomPubkey(), dHash)).to.throw(ERROR_MESSAGES.INVALID_POWPEG_PUBLIC_KEYS);
-    });
-
-    it('should return a valid flyover redeem script', () => {
-        let redeemScript = redeemScriptParser.getFlyoverRedeemScriptFromPublicKeys(publicKeys, dHash).toString('hex');
-        checkPubKeysIncludedInRedeemScript(publicKeys, redeemScript);
-        expect(redeemScript.indexOf(dHash)).to.be.above(0);
+    it('should return a valid p2sh erp redeem script', () => {
+        const p2shErpRedeemScript = redeemScriptParser.getP2shErpRedeemScript(publicKeys, p2shErpBtcPublicKeys, csvValue).toString('hex');
+        validateErpRedeemScriptFormat(p2shErpRedeemScript, publicKeys, p2shErpBtcPublicKeys);
```

Comment:
```suggestion
        validateP2shErpRedeemScriptFormat(p2shErpRedeemScript, publicKeys, p2shErpBtcPublicKeys);
```

Shouldn't be this renamed?

Reply:
Resolved